ups/lsst.table

```diff
@@ -1,7 +1,6 @@
 setupRequired(scons)
 setupRequired(sconsUtils)
 
-envPrepend(PATH, ${PRODUCT_DIR}/bin)
 envPrepend(EUPS_PKGROOT, http://sw.lsstcorp.org/eupspkg, |)
 
 # Deprecated: here for backwards compatibility, but discouraged from using
```

Comment:
I'm fairly confident that `LSST_HOME` from this package isn't being relied on but I'm not sure.  Leaving it doesn't seem to hurt for the moment but I would certainly like to get rid of it.